existing-communities/README.md

```diff
@@ -1 +1,64 @@
 # Existing Communities
+
+## What techniques are currently used to involve a community with software development?
+There are many projects that are build with the help of an online community. The largest projects include the development of the Linux kernel[^linuxrepo], the Apache project[^apacherepo] and the Mozilla browser[^mozillarepo]. To include the online communities during the development of such online software it is necessary to keep everyone up to date and informed about the current state of the project. In short, the means needed can be divided into five categories: communication, version control, issue tracking, testing and package management `citation? wiki: https://en.wikipedia.org/wiki/Open-source_software_development`. For each of these categories will be invested what techniques are currently used by software development projects.
+
+### Communication
+Communication is perhaps the most important factor when it comes to collaborative software development. Much communication is traditionally done via e-mail, instant messaging, phone, web meetings and groupware {{"thissen2007communication" | cite}}. However as of late new, more enhanced tools like Slack[^slackwebsite] have emerged. With for example Slack it is possible to integrate updates from other development tools such as GitHub and make the visible to the whole team.
```

Comment:
`With for example Slack it is possible to integrate updates from other development tools such as GitHub and make the visible to the whole team.`

The start of this sentence is a bit weird, I would rephrase it.